src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,19 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
-            user = authenticationService.login("bob", "1234");
+            user = authenticationService.login("bob", "124");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Authentication attempt failed", e);
```

Comment:
```suggestion
            logger.warn("Authentication attempt failed", e);
```
коли юзер увів неправильні дані, це не помилка роботи програми, тому тут краще error не використовувати